src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -0,0 +1,169 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.github.lindenb.jbwa.jni.AlnRgn;
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.CigarElement;
+import htsjdk.samtools.CigarOperator;
+import htsjdk.samtools.TextCigarCodec;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.CigarUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.List;
+import java.util.Objects;
+
+class AlignmentRegion {
+
+    final String contigId;
+    final String breakpointId;
```

Comment:
I think it makes sense to document that this "breakpointId" might by `null` (when constructed with GATKRead) so accessors should have a null-check, considering it has package-access level.